src/modules/dashboard/components/ColonyMembers/ColonyMembers.tsx

```diff
@@ -122,18 +149,47 @@ const ColonyMembers = () => {
           )}
         </div>
         <aside className={styles.rightAside}>
-          <Button
-            appearance={{ theme: 'blue' }}
-            text={MSG.editPermissions}
-            onClick={handlePermissionManagementDialog}
-            disabled={
-              !isSupportedColonyVersion ||
-              !isNetworkAllowed ||
-              !hasRegisteredProfile ||
-              !colonyData?.processedColony?.isDeploymentFinished ||
-              mustUpgradeOneTx
-            }
-          />
+          <div className={styles.actionButtons}>
+            <Button
+              appearance={{ theme: 'blue' }}
+              text={MSG.editPermissions}
+              onClick={handlePermissionManagementDialog}
+              disabled={
+                !isSupportedColonyVersion ||
+                !isNetworkAllowed ||
+                !hasRegisteredProfile ||
+                !colonyData?.processedColony?.isDeploymentFinished ||
+                mustUpgradeOneTx
+              }
+            />
+          </div>
+          {userCanBan && (
+            <div className={styles.actionButtons}>
+              <Button
+                appearance={{ theme: 'blue' }}
+                text={MSG.banAddress}
+                onClick={() =>
+                  openToggleBanningDialog({
+                    colonyAddress: colonyData?.processedColony?.colonyAddress,
+                  })
+                }
+              />
+            </div>
+          )}
+          {userCanBan && (
+            <div className={styles.actionButtons}>
+              <Button
+                appearance={{ theme: 'blue' }}
+                text={MSG.unbanAddress}
+                onClick={() =>
+                  openToggleBanningDialog({
+                    isBanning: false,
+                    colonyAddress: colonyData?.processedColony?.colonyAddress,
+                  })
+                }
+              />
+            </div>
+          )}
```

Comment:
not sure if i correctly understand you have "userCanBan" twice, so we want display both buttons?

Reply:
I don't really understand your question. `userCanBan` is checked twice - for the html & css structure (not to recreate some additional divs & repeat the css).